scripts/test.sh

```diff
@@ -10,6 +10,8 @@ cat example/datasets/create_model_db.sql | mysql -uroot -proot
 
 export SQLFLOW_TEST_DB=mysql
 
+python -c "import sqlflow_models"
```

Comment:
Is this a leftover for debugging?

Reply:
It's useful to leave it here to ensure `sqlflow_models` is installed, or the following `go test` is not going to output a meaningful error message.

Adding pretty debug information when training script fails seems another work to do, I'll create an issue if needed.